sdk/core/azure-core-management/src/main/java/com/azure/core/management/implementation/polling/PollingState.java

```diff
@@ -78,7 +78,7 @@ public static PollingState create(SerializerAdapter serializerAdapter,
             && httpMethod != HttpMethod.POST
             && httpMethod != HttpMethod.DELETE) {
             throw new IllegalArgumentException("Long-running-operation supported only"
-                + "for PUT, PATCH, POST or DELETE verb.");
+                + " for PUT, PATCH, POST or DELETE verb.");
```

Comment:
I actually not sure whether ARM forbit LRO in GET.

I've also checked with some other languages like python and JS, they did not block LRO on GET.

Let me know if anyone has info.